requirements/pip.txt

```diff
@@ -4,11 +4,11 @@
 #
 #    make upgrade
 #
-wheel==0.37.0
+wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==21.3.1
+pip==22.1.2
     # via -r requirements/pip.in
-setuptools==59.5.0
+setuptools==62.4.0
```

Comment:
`pip.in` needs to have constraints file check in it so it can keep the `setuptools` version pinned.